sound/soc/intel/boards/sof_sdw_maxim.c

```diff
@@ -137,6 +138,14 @@ static int mx8373_sdw_late_probe(struct snd_soc_card *card)
 	return snd_soc_dapm_sync(dapm);
 }
 
+static const struct snd_soc_ops max_98363_sdw_ops = {
+	.startup = sdw_startup,
+	.prepare = sdw_prepare,
+	.trigger = sdw_trigger,
+	.hw_free = sdw_hw_free,
+	.shutdown = sdw_shutdown,
+};
```

Comment:
Humm, one more open @udaymb @bardliao 

Do we really need this structure which is completely generic? We already set the default ops with 
````c
static const struct snd_soc_ops sdw_ops = {
	.startup = sdw_startup,
	.prepare = sdw_prepare,
	.trigger = sdw_trigger,
	.hw_free = sdw_hw_free,
	.shutdown = sdw_shutdown,
};

	init_dai_link(dev, dai_links + *link_index, (*link_id)++, name,
			      playback, capture,
			      cpus + *cpu_id, cpu_dai_num,
			      codecs, codec_num,
			      NULL, &sdw_ops);
````
so is this needed?

Reply:
> Humm, one more open @udaymb @bardliao
> 
> Do we really need this structure which is completely generic? We already set the default ops with
> 
> ```c
> static const struct snd_soc_ops sdw_ops = {
> 	.startup = sdw_startup,
> 	.prepare = sdw_prepare,
> 	.trigger = sdw_trigger,
> 	.hw_free = sdw_hw_free,
> 	.shutdown = sdw_shutdown,
> };
> 
> 	init_dai_link(dev, dai_links + *link_index, (*link_id)++, name,
> 			      playback, capture,
> 			      cpus + *cpu_id, cpu_dai_num,
> 			      codecs, codec_num,
> 			      NULL, &sdw_ops);
> ```
> 
> so is this needed?

Ah, yes, max_98363_sdw_ops is completely generic. We don't need to overwrite the ops. @udaymb

Reply:
Yes i verified this and its not required - hence removed it